jp2_online/settings/base.py

```diff
@@ -177,3 +177,10 @@
 STATIC_URL = '/static/'
 
 LOGIN_URL = 'tosp_auth:login'
+
+EMAIL_HOST = 'smtp.sendgrid.net'
+EMAIL_HOST_USER = 'isjp2.thelma'
+EMAIL_HOST_PASSWORD = 'juanPablo2'
```

Comment:
On a better thought, haha yes, change this to env variable and just send it through slack. Not a good idea to have this in the code.